release_creation/install.sh

```diff
@@ -0,0 +1,4 @@
+pip install -r release_creation/requirements.txt \
```

Comment:
Is `release_creation/requirements.txt` generated at some point prior to this call? Or should this be one of the versioned requirements.txt files under `requirements`?